### PR TITLE
Avoid ClassNotFoundException

### PR DESCRIPTION
**Pull request by jponge**
`wildfly-elytron-x500-cert` has a dependency on `jboss-logging`.

### CODE REVIEW

**Comment by mocenas** [comment]
LGTM

File: api/src/main/java/com/redhat/insights/tls/PEMSupport.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public SSLContext createTLSContext(Path certificatePath, Path keyPath) {
       byte[] certBytes = Files.readAllBytes(certificatePath);
       byte[] keyBytes = Files.readAllBytes(keyPath);
       return createTLSContext(certBytes, keyBytes);
-    } catch (IOException err) {
+    } catch (Throwable err) {
```
Comment (ehsavoie) [inline api/src/main/java/com/redhat/insights/tls/PEMSupport.java:55]: Why Throwable and not Exception ?

Comment (jponge) [reply api/src/main/java/com/redhat/insights/tls/PEMSupport.java:55]: It's a habit from Mutiny where we catch `Throwable` 👍 
Fixing it

**Review by ehsavoie — APPROVED** [review]

**Review by kittylyst — APPROVED** [review]
LGTM. 🚢

**Pull request merged**